### ISSUE TITLE
Create Person

### ISSUE THREAD

**Issue by RobbSadler** [RobbSadler/ILCDirectory#2]
Add a "new" icon (maybe a plus sign) with alt text that says "Create Person" that adds a person row, and then initiates the dialog for editing that row. Once saved, the row should be the selected row on the Person page, with that id selected as the search term (this will be a separate issue).
